base/src/main/java/com/smartdevicelink/proxy/rpc/ShowConstantTbtResponse.java

```diff
@@ -46,19 +46,27 @@
  */
 public class ShowConstantTbtResponse extends RPCResponse {
 
+    /**
+     * Constructs a new ShowConstantTBTResponse object
+     */
     public ShowConstantTbtResponse() {
         super(FunctionID.SHOW_CONSTANT_TBT.toString());
     }
 
+    /**
```

Comment:
@kostyaBoss I would delete the comment block on lines 56-60.  The idea is that nothing should be changed in this file.